src/main/java/com/assignment/Assignment9.java

```diff
@@ -0,0 +1,14 @@
+package com.assignment;
+public class Assignment9 {
+    public static void evenNumber() {
```

Comment:
Lets modify this function, as :
lets this function accept two parameters - upper and lower limit i.e the range to find the even numbers.
and let this function return the even numbers in an arrayList